packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -424,14 +447,14 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
                     <Box>
                       <Heading type="title2">{label}</Heading>
                     </Box>
-                    <ModalCloseButton onClick={handleClose} title="Close" />
+                    <ModalCloseButton onClick={_ev => handleClose()} title="Close" />
                   </Stack>
                 )}
                 {input}
               </StyledModalHeader>
               <StyledModalSection>{dropdown}</StyledModalSection>
               <ModalFooter flex="100%">
-                <Button type="secondary" fullWidth onClick={handleClose}>
+                <Button type="secondary" fullWidth onClick={_ev => handleClose()}>
```

Comment:
can you make a handleClose as a carried function that receives event? So you do not need to call it like that on multiple places

Reply:
For the same reason mentioned before, I can't. However, I was able to merge the function handled on outsideClick and the ones called onClick into one, because they both expect events as arguments (even though they do nothing with them) and they should both call `handleClose` without any parameter.

Reply:
I meant it something like that: 

*const handleClose = (selection?: Option | null) => (_ev?: unkown) =>* 

then you can use it just like: 

*onClose={handleClose()}*

or *handleClose(option)*

Reply:
tried that with carried function, works ok, but I am ok with your approach also, so resolving